testing/jormungandr-integration-tests/src/common/jcli/api/certificate.rs

```diff
@@ -27,9 +27,9 @@ impl Certificate {
             .as_single_line()
     }
 
-    pub fn new_vote_tally<S: Into<String>>(self, vote_plan_id: S) -> String {
+    pub fn new_vote_tally<S: Into<String>>(self, vote_plan_id: S, tally_type: S) -> String {
```

Comment:
@dkijania are these changes ok? or do you think it should have a custom type somewhere?

Reply:
A custom type would be more disciplined. Could reuse from jormungandr-lib?

Reply:
We could use `PayloadTypeDef`..